deluge/core/torrentmanager.py

```diff
@@ -693,7 +693,7 @@ def _save_state(self):
                 os.fsync(_file.fileno())
         except (OSError, cPickle.PicklingError) as ex:
             log.error("Unable to save %s: %s", filename, ex)
-            return
+            return False
```

Comment:
Umm...no
I checked the return value during testing, but it's not needed for the PR. Would be reasonable to return a value or success/failure? Not strictly used for anything at the moment tho.